packages/frontend/src/components/BigNumberConfig/index.tsx

```diff
@@ -24,19 +31,14 @@ export const BigNumberConfigPanel: React.FC = () => {
     const [isOpen, setIsOpen] = useState(false);
     const disabled = !selectedField;
 
-    const styleOptions = [
-        { value: '', label: 'none' },
-        { value: NumberStyle.THOUSANDS, label: 'thousands (K)' },
-        { value: NumberStyle.MILLIONS, label: 'millions (M)' },
-        { value: NumberStyle.BILLIONS, label: 'billions (B)' },
-    ];
     return (
         <Popover2
             disabled={disabled}
             content={
-                <>
-                    <InputWrapper label="Select field">
+                <InputWrapper>
+                    <FormGroup labelFor="bignumber-field" label="Select field">
```

Comment:
is this the actual fix ?

Reply:
no no. this is just a styling fix. 

old:
<img width="384" alt="CleanShot 2022-10-10 at 19 44 49@2x" src="https://user-images.githubusercontent.com/962095/194905257-fc4c9990-e113-4719-a97d-b989d7dd6aa3.png">

new:
<img width="410" alt="CleanShot 2022-10-10 at 19 48 51@2x" src="https://user-images.githubusercontent.com/962095/194905970-b0e0b7de-4aad-462e-bfde-6c755147dc93.png">

Reply:
@IrakliJani did you notice the InputWrapper is a styled component of a FormGroup? perhaps we just need to change the styled component

Reply:
@ZeRego 

yea, InputWrapper is used in all over config files.
besides, I don't understand what's going in its styles
```
    margin: 1.357em 0 0;
    & label.bp4-label {
        display: inline-flex;
        gap: 0.214em;
        color: ${Colors.DARK_GRAY1};
        font-weight: 600;
    }
```
I better stick with Blueprint's exposed FormGroup instead